src/assets/locales/en.json

```diff
@@ -9,23 +9,34 @@
       "the_course_has_no_lessons_yet": "The course has no lessons yet!",
       "back_to_enrolled_courses": "Back To Enrolled Courses",
       "youve_completed_the_course": "You've completed the course",
-      "we_could_not_find_any_courses. thanks for you patient": "We could not find any courses. thanks for you patient"
+      "we_could_not_find_any_courses. thanks for you patient": "We could not find any courses. thanks for you patient",
```

Comment:
When would this appear?

Reply:
<img width="653" alt="image" src="https://github.com/Focus-Bear/online_course/assets/11420548/e5e56d4f-1088-47df-8940-feacbfbea516">